docs/throughput-anomaly-detection.md

```diff
@@ -69,6 +69,34 @@ $ theia throughput-anomaly-detection run --algo "ARIMA"
 Successfully started Throughput Anomaly Detection job with name tad-1234abcd-1234-abcd-12ab-12345678abcd
 ```
 
+Throughput Anomaly Detection also provide support for aggregated throughput
+anomaly detection.
+There are three different types of aggregations that are included.
+
+- `external` : Aggregated flow for inbound traffic to external IP,
+  user could provide external-IP using `external-ip` argument for further
+  filtering.
+- `pod`: Aggregated flow for inbound/outbound pod traffic.
+- `svc`: Aggregated flow for traffic to service port, user could
+  provide a destination port name using `svc-name-port` argument for
+  further filtering.
+
+For aggregated flow `pod`, user can provide the following filter arguments.
+
+- `pod-label`: The argument aggregates inbound/outbound traffic using pod
+  labels.
+- `pod-name`: The argument aggregates inbound/outbound traffic using pod name.
+- `pod-namespace`: The argument aggregates inbound/outbound traffic using
+  podnamespace. However, this argument only works as a combination to any of
+  the above two arguments and can not be used alone.
+
+All the above aggregated flow are executed using the following command:
```

Comment:
It is a little bit confused for me to "execute the aggregated flow". Maybe "To start an aggregated throughput anomaly detection, please run the following command"?